sql/core/src/main/scala/org/apache/spark/sql/execution/exchange/EnsureRequirements.scala

```diff
@@ -150,10 +150,15 @@ case class EnsureRequirements(
         //   A: (No_Exchange, 100) <---> B: (Exchange, 120)
         // it's better to pick A and change B to (Exchange, 100) instead of picking B and insert a
         // new shuffle for A.
```

Comment:
Can we update this code comment w.r.t. the new change?

Reply:
👌